### DISCUSSION TITLE
URI edits

### DISCUSSION THREAD

**Discussion by Grumpylo** [dani-garcia/vaultwarden#1517] [Q&A]
I was hoping to use the auto fill so I went to my Vault to look at the URI listings.  I tested one of my sites and had no trouble getting logged in.  However when I looked at the URI information, I noticed that it began to change without my input!  Another site (reddit.com) INPUT ITS OWN INFORMATION!  Very weird.  I am lost with this problem.

**Comment by jjlin** [comment]
Maybe you logged into reddit.com using that entry and selected the "Auto-fill and save" option, which saves the association between the site and that entry.

**Reply by Grumpylo** [reply]
That could be a possible.  I have been looking since you gave me the hint but I don't find, so far, any method of rectification.  If you have more suggestions, I am all eyes.  Thanks.

**Reply by jjlin** [reply]
I'm not completely clear on what you're asking, but if you're asking how to remove the URI, you can just edit the entry and remove it.

**Reply by Grumpylo** [reply]
Thanks.  However, when I do that edit, it makes no difference.  
So after trying it on five of the URI's I reckon that I will just delete everything and start over.  
Thanks, again.
